.github/workflows/docker-ci.yml

```
@@ -296,17 +329,40 @@ jobs:
-            VERSION_ID=$(gh api \
-              "/user/packages/container/img2num-dev/versions" \
-              --paginate \
-              --jq ".[] | select(.metadata.container.tags[] == \"${TAG}\") | .id")
-            if [[ -n "$VERSION_ID" ]]; then
-              echo "Deleting version $VERSION_ID (tag: $TAG)"
-              gh api --method DELETE "/user/packages/container/img2num-dev/versions/${VERSION_ID}"
-            else
-              echo "No version found for tag: $TAG"
-            fi
-          done
+  set +e
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)